README.md

```diff
@@ -58,6 +60,12 @@ Environment variables:
 * `TRELLO_TEST_PASSWORD`
     * required
     * Password to use while running cukes
+* `INTEGRATION_KEY`
+    * required
+    * same as your `PUBLIC_KEY` up above, but is used for integration specs
+* `INTEGRATION_TOKEN`
```

Comment:
This seems to be an awkward environment variable to get. It would be easier to just read a public board as described in the token docs: https://trello.com/app-key#.